### ISSUE TITLE
 does not provide an export named 'BrowserQRCodeReader'

### ISSUE THREAD

**Issue by abidhkm** [zxing-js/library#139]
I'm getting error on console when running the project .
`localhost/:1 Uncaught SyntaxError: The requested module '/node_modules/@zxing/library/esm5/index.js' does not provide an export named 'BrowserQRCodeReader'`

The strangest thing is the same code works in stackblitz sample code but not in local environment
link for stackblitz: [link](https://stackblitz.com/edit/naxw5o-qtyvc4?file=custom-greeting.js)
link for project in github to run in local environment : [link](https://github.com/abidhkm/qr-dode-reader-zxing)

**Comment by odahcam** [comment]
- Are you using NPM, or Yarn, or what?
- What's your Node version?
- What's the library version you got installed? (please check package-lock.json or yarn.lock)
- Are you on Windows or what OS?

Hint: your project is almost empty, try a fresh start, I did it and the lib was imported normally.

![image](https://user-images.githubusercontent.com/3942006/53305750-04355580-3864-11e9-96ec-7680ab8c5695.png)

PS.: nice to see you're using @open-wc :)

**Comment by rjsteinert** [comment]
I'm hitting this issue trying to include in a Polymer 3 generated project.

**Comment by rjsteinert** [comment]
I forked this repo, [tweaked the tsconfig.json to make an ES6 build](https://github.com/rjsteinert/zxing-js/commit/a693573cd552284015b624262e8c5c63444b5476), and published it on npm as `zxing-js-es6`.  Right now the following works for me in my Polymer project...

Install:
```
npm install --save zxing-js-es6
```

Import:
```
import { BrowserMultiFormatReader } from 'zxing-js-es6/es6';
```

It would be great if we could get es6 build target builds into the release workflow in this repo. Unfortunately I'm not familiar with the best approach here with TS.

![no idea](http://www.dumpaday.com/wp-content/uploads/2013/05/i-have-no-idea-what-Im-doing-meme-4.jpg)

**Comment by odahcam** [comment]
We would have to change some targets or even create another build output, as I did to get this working about a year ago. I find it totally valid to aim for a ES6 build output and an AMD one, so we can get the most optimised code for each.

**Comment by odahcam** [comment]
I've published a new version (v0.15.2) with ES6 module output.
